src/testHelpers/assertions.js

```diff
@@ -0,0 +1,5 @@
+const assertFirstChildIsNull = container => {
+  expect(container.firstChild).toBeNull();
```

Comment:
I don't think this line needs an abstraction. Personally I think this line of code is more readable than the function name anyway and I think it's an abstraction that other devs won't discover and won't look for or need

Reply:
If you think it helps tidy up your tests then you could just colocate it with your tests

Reply:
I think you're right - happy to remove it.